addons/web_editor/static/shapes/Origins/02_001.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1400 1140">
```

Comment:
The purpose was to revert this shape, in this way the color preset work better. I also edited the height of the top rectangle (always 33% now) and his position which is more logical to use.
![demo](https://user-images.githubusercontent.com/80679690/121182744-b04bb080-c863-11eb-8b2d-a67e97dea356.jpg)

Reply:
Did not mean to comment this shape, oops... I was talking about `Bold/12_001.svg`

Reply:
As you can see, the new version works better with the color preset :)
![bold_12](https://user-images.githubusercontent.com/80679690/121202556-ad59bb80-c875-11eb-979a-e20e0564e257.jpg)
.

Reply:
I cannot see :grimacing:  Your before/after is what happens in master with or without using a dark background preset. It's the same with your new shape when the text is on the left... :thinking:

Reply:
My example was maybe not very clear 😅 .

Here's another one, based on the same color configuration: the purpose is to use the color preset in the biggest part of the snippet. In this way, the result matches better and it's not needed to do an advanced configuration with the color of the shape to get a good result. 

Tell me if it's still not clear 😉 
![demo](https://user-images.githubusercontent.com/80679690/121319152-37546380-c90c-11eb-9383-83d39613d00d.jpg)

Reply:
I understand but you keep using *the* problematic snippet ^^ Use text-image instead of image-text and the problem is reversed :grimacing: Anyways, I am good with the "there is 10% more preset color appearing" explanation :grimacing: